source/ncnn.js

```diff
@@ -287,6 +294,27 @@ ncnn.Node = class {
                 this._weight(blobReader, 'bias', [ bias_data_size ], 'float32');
                 break;
             }
+            case 'BinaryOp': {
```

Comment:
Is there a way to improve this that does not require changing `type` semantics? For example, always show activation type even if "Show Attributes" is disabled?